src/internal/Subscriber.ts

```diff
@@ -74,10 +76,12 @@ export class Subscriber<T> extends Subscription implements Observer<T> {
    * @param {any} [err] The `error` exception.
    * @return {void}
    */
-  error(err?: any): void {
-    if (!this.isStopped) {
+  error(error?: any): void {
+    if (this.isStopped) {
+      handleStoppedNotification({ kind: 'E', error }, this);
```

Comment:
These could use `errorNotification(error)` or `nextNotification(value)`, etc. It will minify nicer.